src/gui/plugins/component_inspector/Light.qml

```diff
@@ -356,360 +345,85 @@ Rectangle {
-
-            Text {
-              id : bSpecularText
-              text: ' B'
-              leftPadding: 5
-              color: Material.theme == Material.Light ? "#444444" : "#bbbbbb"
-              font.pointSize: 12
-              anchors.centerIn: parent
-            }
-          }
-          Item {
-            Layout.fillWidth: true
-            height: 40
-            Loader {
-              id: bSpecularLoader
-              anchors.fill: parent
-              property double numberValue: model.data[2]
-              sourceComponent: sliderZeroOne
-              onLoaded: {
-                bSpecularItem = bSpecularLoader.item
-              }
-            }
-          }
-          Rectangle {
-            color: "transparent"
-            height: 40
-            Layout.preferredWidth: aSpecularText.width + indentation*3
-            Loader {
-              id: loaderSpecularA
-              width: iconWidth
-              height: iconHeight
-              y:10
-              sourceComponent: plotIcon
-            }
-            Component.onCompleted: loaderSpecularA.item.componentInfo = "specularA"
-
-            Text {
-              id : aSpecularText
-              text: ' A'
-              leftPadding: 5
-              color: Material.theme == Material.Light ? "#444444" : "#bbbbbb"
-              font.pointSize: 12
-              anchors.centerIn: parent
-            }
-          }
-          Item {
-            Layout.fillWidth: true
-            height: 40
-            Loader {
-              id: aSpecularLoader
-              anchors.fill: parent
-              property double numberValue: model.data[3]
-              sourceComponent: sliderZeroOne
-              onLoaded: {
-                aSpecularItem = aSpecularLoader.item
-              }
-            }
-          }
-        }
-        RowLayout {
-          Layout.alignment: Qt.AlignHCenter
-          Button {
-            Layout.alignment: Qt.AlignHCenter
-            id: specularColor
-            text: qsTr("Specular Color")
-            onClicked: colorDialog.open()
-            ColorDialog {
-              id: colorDialog
-              title: "Choose a specular color"
-              visible: false
-              onAccepted: {
-                rSpecularLoader.item.value = colorDialog.color.r
-                gSpecularLoader.item.value = colorDialog.color.g
-                bSpecularLoader.item.value = colorDialog.color.b
-                aSpecularLoader.item.value = colorDialog.color.a
-                sendLight()
-                colorDialog.close()
-              }
-              onRejected: {
-                colorDialog.close()
-              }
-            }
-          }
-        }
         RowLayout {
+          // Color
           Text {
             Layout.columnSpan: 6
-            text: "      Diffuse"
+            text: "Color"
             color: "dimgrey"
-            width: margin + indentation
+            font.bold: true
```

Comment:
I think removing the bold font might cause trouble for users to tell between the titles and the contents.

Reply:
Ok, then can you also bold `Direction` and `Spot features` so that they are all consistent? 

Also, can you add a little spacing so that the plot icons aren't at the very edge of the plugin border?

![image](https://user-images.githubusercontent.com/8602001/184456751-94f6670d-5851-45d1-8163-7e3df383514f.png)